apps/console/src/features/applications/components/settings/certificate/application-certificate-wrapper.tsx

```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```

Comment:
Can we keep the year unchanged?
```suggestion
 * Copyright (c) 2021, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
```